spec/errors.md

```diff
@@ -335,3 +307,65 @@ or an internally inconsistent set of options.
 > Your {$field} is {$id :get field=$field}
 > ```
 
+### Bad Operand
+
+A **_<dfn>Bad Operand</dfn>_** error is an error that occurs when
+an _operand_ provided to a _function_ during _function resolution_ does not match one of the
```

Comment:
I'd break this up a bit to provide some wiggle room for any cases we forgot and to relax when it happens. "... during function resolution..." is too specific, since it would potentially preclude a tool from detecting it during parsing, etc.

```suggestion
A **_<dfn>Bad Operand</dfn>_** error is any error that occurs due to the content or format of the _operand_,
such as when the _operand_ provided to a _function_ during _function resolution_ does not match one of the